### PR TITLE
Fix Orca error message: Orca supports Rollup #117665973

### PR DESCRIPTION
**Pull request by vraghavan78**
Fixing the error message.

### CODE REVIEW

File: src/backend/gpopt/translate/CTranslatorUtils.cpp [hunk 1]
```diff
@@ -1583,9 +1585,9 @@ CTranslatorUtils::PdrgpbsGroupBy
 		return PdrgpbsRollup(pmp, pgrcl, ulCols, phmululGrpColPos, pbsGrpCols);
 	}
 
-	if (GROUPINGTYPE_GROUPING_SETS != pgrcl->groupType)
+	if (GROUPINGTYPE_CUBE == pgrcl->groupType)
 	{
-		GPOS_RAISE(gpdxl::ExmaDXL, gpdxl::ExmiQuery2DXLUnsupportedFeature, GPOS_WSZ_LIT("Rollup and cube"));
+		GPOS_RAISE(gpdxl::ExmaDXL, gpdxl::ExmiQuery2DXLUnsupportedFeature, GPOS_WSZ_LIT(“Cube”));
```
Comment (hornn) [inline src/backend/gpopt/translate/CTranslatorUtils.cpp]: the quote marks look off. should be `"` and not `“`

**Comment by vraghavan78** [comment]
Thanks @hornn

**Comment by changleicn** [comment]
@vraghavan78 GA build was cut yesterday. so this can only be after 2.0. the master branch will be open for commit today.

**Comment by liming01** [comment]
@vraghavan78 , merged, please close this PR. Thanks.

**Pull request closed without merging**